### PR TITLE
Apply minor fixes to README

### PR DESCRIPTION
**Pull request by geoand**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,8 +20,7 @@ or JAX-RS resources.
 
 ### Prerequisites 
 
-Unfortunately, right now, there are no release of this project, so you need to build it by yourself. The build process
- requires Apache Maven and Java 8+.  You need to build the SmallRye Reactive Stream Operators implementation first:
+The build process requires Apache Maven and Java 8+. You need to build the SmallRye Reactive Stream Operators implementation first:
```
Comment (cescoffier) [inline README.md]: Actually... this is not true anymore :-)

Comment (cescoffier) [reply README.md]: The only requirements are Apache Maven and Java 8+.

Comment (geoand) [reply README.md]: Done :)

**Comment by cescoffier** [comment]
Thanks!

**Comment by geoand** [comment]
:+1:

**Pull request merged**